spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -535,17 +536,25 @@
                   },
                 },
               },
-              pii_from_doc: {
-                first_name: first_name,
-                last_name: last_name,
-                address1: address1,
-                state: state,
-                state_id_type: state_id_type,
-                dob: dob,
-                state_id_jurisdiction: jurisdiction,
-                state_id_number: state_id_number,
-                zipcode: zipcode,
-              },
+              pii_from_doc: Pii::StateId.new(
+                **{
+                  first_name: first_name,
+                  last_name: last_name,
+                  middle_name: nil,
+                  address1: address1,
+                  state: state,
+                  state_id_type: state_id_type,
+                  dob: dob,
+                  state_id_jurisdiction: jurisdiction,
+                  state_id_number: state_id_number,
+                  zipcode: zipcode,
+                  address2: nil,
+                  city: nil,
+                  state_id_expiration: nil,
+                  state_id_issued: nil,
+                  issuing_country_code: nil,
+                },
+              ),
```

Comment:
this `**` splat is redundant (unless we are calling `.compact` on the hash or something), we can just pass keyword args directly:

```suggestion
              pii_from_doc: Pii::StateId.new(
                first_name: first_name,
                last_name: last_name,
                middle_name: nil,
                address1: address1,
                state: state,
                state_id_type: state_id_type,
                dob: dob,
                state_id_jurisdiction: jurisdiction,
                state_id_number: state_id_number,
                zipcode: zipcode,
                address2: nil,
                city: nil,
                state_id_expiration: nil,
                state_id_issued: nil,
                issuing_country_code: nil,
              ),
```